File: Programmers/Java/Lv.0/[Lv.0] 옷가게 할인 받기/[Lv.0] 옷가게 할인 받기_00.java
class Solution {
    public int solution(int price) {
        int answer = 0;

        // 할인가 = price * 할인율(100 - 할인 %)
        if (price >= 500000) {
            answer = (int) (price *= 0.8);
        } else if (price >= 300000) {
            answer = (int) (price *= 0.9);
        } else if (price >= 100000) {
            answer = (int) (price *= 0.95);
        } else {
            answer = price;
        }

        return answer;
    }
}

File: Programmers/Python/Lv.0/[Lv.0] 옷가게 할인 받기/[Lv.0] 옷가게 할인 받기_00.py
def solution(price):
    # 할인가 = price * 할인율(100 - 할인%)
    if price >= 500000:
        price *= 0.8
    elif price >= 300000:
        price *= 0.9
    elif price >= 100000:
        price *= 0.95

    return int(price)
